backend/btrixcloud/models.py

```diff
@@ -849,3 +864,53 @@ class UserDB(User, fastapi_users_models.BaseUserDB):
     """
 
     invites: Dict[str, InvitePending] = {}
+
+
+# ============================================================================
+
+### WEBHOOKS ###
+
+
+# ============================================================================
+class WebhookNotificationBody(BaseModel):
+    """Base POST body model for webhook notifications"""
+
+    downloadUrls: Optional[List] = None
+
+
+# ============================================================================
+class ArchivedItemCreatedBody(WebhookNotificationBody):
+    """Webhook notification POST body for when archived item is created"""
+
+    itemId: str
+
+
+# ============================================================================
+class CollectionItemAddedRemovedBody(WebhookNotificationBody):
+    """Webhook notification POST body for when item is added to or removed from collection"""
+
+    collectionId: str
+    itemIds: List[str]
+    type: str = "added"
+
+
+# ============================================================================
+class WebhookEventType(str, Enum):
+    """Webhook Event Types"""
+
+    ARCHIVED_ITEM_CREATED = "archived-item-created"
```

Comment:
Perhaps these ids could match the attribute names in `OrgWebHookUrls`?
This could make it a bit less verbose / easier to extend in places.